HHS/Head-Start-TTADP
# Remove total grants statistics from the TTA overview widget plus remove topics rename from cron - prod

## Description of change
Removes the total grants statistics from the display on the TTA Overview widget. Updates the text for non-grantee statistics to be "Non-grantee entities served". Fixes the export of reports - adding filtering by regions.


## How to test
- Go to Staging - https://tta-smarthub-staging.app.cloud.gov/
- verify that (of x) is not showing under grants on the TTA overview widget.
- verify that the non-grantee statistic has a changed text.
- the display of the TTA overview widget is now showing for all, admins and non-admins. This can be verified by removing of the  admin permission from yourself in the admin UI, going to activity reports and then refreshing the page. The widget should still be visible. (Please slack me to enable the admin permissions back).

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-143
* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-243


## Checklists

### Every PR


- [ n/a] API Documentation updated
- [ n/a] Boundary diagram updated
- [ n/a] Logical Data Model updated
- [ n/a] [Architectural Decision Records](https://adr.github.io/) written for major infrastructure decisions

### Production Deploy


### After merge/deploy
